client/my-sites/checkout/src/lib/get-flow-plan-features.ts

```diff
@@ -2,13 +2,118 @@ import {
 import { getPlanFeaturesObject } from 'calypso/lib/plans/features-list';
-import {
-	getHighlightedFeatures,
-	getPlanFeatureAccessor,
-} from 'calypso/my-sites/plan-features/util';
+
+const newsletterFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return flowName === NEWSLETTER_FLOW && plan.getNewsletterSignupFeatures;
+};
+
+const linkInBioFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return isLinkInBioFlow( flowName ) && plan.getLinkInBioSignupFeatures;
+};
+
+const hostingFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return isAnyHostingFlow( flowName ) && plan.getHostingSignupFeatures?.( plan.term );
+};
+
+const blogOnboardingFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return isBlogOnboardingFlow( flowName ) && plan.getBlogOnboardingSignupFeatures;
+};
+
+const signupFlowDefaultFeatures = (
+	flowName: string,
+	plan: IncompleteWPcomPlan,
+	isInVerticalScrollingPlansExperiment: boolean
+) => {
+	if ( ! flowName || isNewsletterOrLinkInBioFlow( flowName ) ) {
+		return;
+	}
+
+	return isInVerticalScrollingPlansExperiment
+		? plan.getSignupFeatures
+		: plan.getSignupCompareAvailableFeatures;
+};
+
+const getPlanFeatureAccessor = ( {
+	flowName = '',
+	isInVerticalScrollingPlansExperiment = false,
+	plan,
+}: {
+	flowName?: string;
+	isInVerticalScrollingPlansExperiment?: boolean;
+	plan: IncompleteWPcomPlan;
+} ) => {
+	return [
+		newsletterFeatures( flowName, plan ),
+		linkInBioFeatures( flowName, plan ),
+		hostingFeatures( flowName, plan ),
+		blogOnboardingFeatures( flowName, plan ),
+		signupFlowDefaultFeatures( flowName, plan, isInVerticalScrollingPlansExperiment ),
+	].find( ( accessor ) => {
+		return accessor instanceof Function;
+	} );
+};
+
+const newsletterHighlightedFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return flowName === NEWSLETTER_FLOW && plan.getNewsletterHighlightedFeatures;
+};
+
+const linkInBioHighlightedFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return isLinkInBioFlow( flowName ) && plan.getLinkInBioHighlightedFeatures;
+};
+
+const hostingHighlightedFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return isLinkInBioFlow( flowName ) && plan.getHostingHighlightedFeatures;
+};
+
+const blogOnboardingHighlightedFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	return isBlogOnboardingFlow( flowName ) && plan.getBlogOnboardingHighlightedFeatures;
+};
+
+const getHighlightedFeatures = ( flowName: string, plan: IncompleteWPcomPlan ) => {
+	const accessor = [
+		newsletterHighlightedFeatures( flowName, plan ),
+		linkInBioHighlightedFeatures( flowName, plan ),
+		hostingHighlightedFeatures( flowName, plan ),
+		blogOnboardingHighlightedFeatures( flowName, plan ),
+	].find( ( accessor ) => {
+		return accessor instanceof Function;
+	} );
+
+	return ( accessor && accessor() ) || [];
+};
+
+export const getPlanDescriptionForMobile = ( {
```

Comment:
Are there any other usages of this function apart from this one?
https://github.com/Automattic/wp-calypso/blob/2289bd822fcadee173e49003a57a412fec6b08f3/client/my-sites/plan-features/index.jsx#L383

If not, we can remove this too. :)

Reply:
Oohh nice catch! :)